4.3/README.md

```diff
@@ -42,9 +44,17 @@ If you want to check that your data import was successful, you can use the API w
 
 The following environment variables are available for configuration:
 
-- `PBF_URL`: Which [OSM extract](#openstreetmap-data-extracts) to download and import. It cannot be used together with `PBF_PATH`. Check [https://download.geofabrik.de](https://download.geofabrik.de)
+- `PBF_URL`: Which [OSM extract](#openstreetmap-data-extracts) to download and import. It cannot be used together with `PBF_PATH`.
+  Check [https://download.geofabrik.de](https://download.geofabrik.de) 
+  Since de DL speed is restrictet at Geofabrik, for importing the full Planet there is a recommended list of Mirrors at the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors).
+  There you can find the folder /planet which contains the Planet-latest.osm.pbf
+  and mostly a /replication folder for the `REPLICATION_URL`.
```

Comment:
I'm not sure I understand these two lines. What does "there" refer to?

Reply:
I change the sentece - the mirror sites where ment to explain where you have to look for the right folders if youre not familar there.

Reply:
i have rewritten the comfusing part.